README.md

```diff
@@ -38,7 +38,7 @@ If you feel like contributing to Nicotine+, there are several ways to help:
 # Where did the name Nicotine come from?
 
 > I was in a geeky mood and was browsing [http://www.bash.org](http://www.bash.org)'s QDB.  
-I stumbled across this quote:  
+I stumbled across [this quote](http://www.bash.org/?10159):  
```

Comment:
Instead of this additional link, the link in the line above might as well point to the specific page, instead of linking to bash.org's home page. Also the statement can be combined onto one line...

```
> I was in a geeky mood and was browsing [bash.org](http://www.bash.org/?10159)'s QDB,  I stumbled across this quote:
```

> I was in a geeky mood and was browsing [bash.org](http://www.bash.org/?10159)'s QDB,  I stumbled across this quote:

Reply:
Yes and that's good, I'm just saying the existing hyperlink should be changed instead of creating a new one.